lib/appsignal/transaction/registry.ex

```diff
@@ -36,11 +36,9 @@ defmodule Appsignal.TransactionRegistry do
     pid = self()
 
     if Appsignal.Config.active?() && registry_alive?() do
-      monitor_reference = GenServer.call(__MODULE__, {:monitor, pid})
+      monitor_reference = Process.monitor(pid)
```

Comment:
Now, this may have to go into a GenServer since it may not be linking the proper process.